rename.rb

```diff
@@ -29,26 +29,20 @@ def usage
   exit 1
 end
 
-old_dirs = []
 Find.find('.') do |path|
-  next unless File.file?(path)
-  next if path =~ /\.git/
-  next if path == './rename.rb'
-
-  # Change content on all files
-  tmp_file = "#{path}.tmp"
-  system(%(sed 's/foreman_plugin_template/#{snake}/g' #{path} > #{tmp_file}))
-  system(%(sed 's/ForemanPluginTemplate/#{camel}/g' #{tmp_file} > #{path}))
-  system(%(sed 's/foremanPluginTemplate/#{camel_lower}/g' #{tmp_file} > #{path}))
-  system(%(rm #{tmp_file}))
+  if File.basename(path) == '.git'
+    Find.prune
+  elsif File.file?(path)
+    system(%(sed -i 's/foreman_plugin_template/#{snake}/g; s/ForemanPluginTemplate/#{camel}/g; s/foremanPluginTemplate/#{camel_lower}/g' #{path}))
```

Comment:
This line should now fix this issue: https://github.com/theforeman/foreman_plugin_template/issues/63

Reply:
I'd like it if it was at least a separate commit for visibility.